dist/index.js

```diff
@@ -440,7 +440,7 @@ function run() {
                             const res = _c;
                             const artifact = res.data.find((artifact) => !artifact.expired && artifact.name === baseReport);
                             if (!artifact) {
-                                yield new Promise((resolve) => setTimeout(resolve, 900)); // avoid reaching the API rate limit
+                                yield new Promise((resolve) => setTimeout(resolve, 500)); // avoid reaching the API rate limit
```

Comment:
Pretty sure that this is just making requests more frequently. We should add an optional bootstrap commit hash which we don't look further than.

Reply:
> Pretty sure that this is just making requests more frequently. We should add an optional bootstrap commit hash which we don't look further than.

Yeah I realized that after it was done. Guess I just lucked out the PR on noir-lang didn't hit the rate limit.

Reply:
As I got a working report now should we do this in a follow-up PR? I can do the change in this PR if we still desire that though.